clusterloader2/pkg/prometheus/manifests/dashboards/network.dashboard.py

```diff
@@ -0,0 +1,120 @@
+#!/usr/bin/env python
+
+# Copyright 2019 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from grafanalib import core as g
+import defaults as d
+
+QUANTILES = [0.99, 0.9, 0.5]
+
+
+def show_quantiles(queryTemplate, quantiles=None, legend=""):
+    quantiles = quantiles or QUANTILES
+    targets = []
+    for quantile in quantiles:
+        q = "{:.2f}".format(quantile)
+        l = legend or q
+        targets.append(g.Target(expr=queryTemplate.format(quantile=q), legendFormat=l))
+    return targets
+
+
+NETWORK_PROGRAMMING_PANEL = [
+    d.Graph(
+        title="SLI: Network programming latency",
+        description=(
+            "NetworkProgrammingLatency is defined as the time it took to "
+            + "program the network - from the time  the service or pod has "
+            + "changed to the time the change was propagated and the proper "
+            + "kube-proxy rules were synced. Exported for each endpoints object "
+            + "that were part of the rules sync."
+        ),
+        targets=show_quantiles(
+            (
+                "quantile_over_time("
+                + "0.99, "
```

Comment:
Just to double check, you really want to write 0.99 here instead of e.g. {quantile}?

Reply:
Yes, this is my intention. SLI is defined for 99th percentile.